specs/AppLifecycle/StateNotifications/AppLifecycle StateNotifications.md

```diff
@@ -239,147 +233,142 @@ The well-known
-| GUID_POWERSCHEME_PERSONALITY      | The active power scheme personality has changed. All power schemes map to one of these personalities.                                                                                                                  | `PowerSchemePersonality`                                               |
-| GUID_SESSION_DISPLAY_STATUS       | The display associated with the application's session has been powered on or off.                                                                                                                                      | Covered by GUID_CONSOLE_DISPLAY_STATE.                                 |
-| GUID_SESSION_USER_PRESENCE        | The user status associated with the application's session has changed.                                                                                                                                                 | `UserPresenceStatus`                                                   |
-| GUID_SYSTEM_AWAYMODE              | The system is entering or exiting away mode.                                                                                                                                                                           | `SystemAwayModeStatus` |
+| Existing GUID                     | Description                                                  | New property and event                                    |
+| --------------------------------- | ------------------------------------------------------------ | --------------------------------------------------------- |
+| GUID_ACDC_POWER_SOURCE            | The system power source has changed.                         | `PowerSourceKind, PowerSourceKindChanged`                 |
+| GUID_BATTERY_PERCENTAGE_REMAINING | The remaining battery capacity has changed.                  | `RemainingChargePercent`, `RemainingChargePercentChanged` |
+| GUID_CONSOLE_DISPLAY_STATE        | The current monitor's display state has change. Apps should stop rendering graphics content when the monitor is off to reduce system power consumption. | _Not implemented_                                         |
+| GUID_GLOBAL_USER_PRESENCE         | The user status associated with any session has changed. This notification is only to services and other programs running in session 0. User-mode applications should register for GUID_SESSION_USER_PRESENCE instead. | _Not implemented_                                         |
+| GUID_IDLE_BACKGROUND_TASK         | The system is busy: now is a good time for apps to perform background or idle tasks that would otherwise prevent the computer from entering an idle state. | `SystemIdleStatus, SystemIdleStatusChanged`               |
```

Comment:
There is no property for this one; just an event.

Reply:
No 'SystemIdleStatus'

Reply:
Fixed